File: Assignment Programs/Assignment33/program1.java
import java.util.*;

class Digit
{
	public int CountEven(int iNo)
	{
		int iCnt = 0, iDigit = 0;
		
		while(iNo != 0)
		{
			iDigit = iNo % 10;
			if(iDigit % 2 == 0)
			{
				iCnt++;
			}
			iNo = iNo / 10;
		}
		return iCnt;
	}
}

class program1
{
	public static void main(String args[])
	{
		int iValue = 0, iRet = 0;
		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter the number");
		iValue = sobj.nextInt();
	
		Digit dobj = new Digit();
		
		iRet = dobj.CountEven(iValue);
	
		System.out.println("Count of even digits from number is : "+iRet);
	}
}

File: Assignment Programs/Assignment4/program1.c
#include<stdio.h>


int MultFact(int iNo)
{
	int iCnt = 0;
	int mul = 1;
	for(iCnt=1;iCnt<=iNo;iCnt++)
	{
		if(iNo%iCnt==0)
		{
			mul = mul*iCnt;
		}
	}
	return mul;
}
int main()
{
	int iValue = 0;
	int iRet = 0;
	printf("Enter number\n");
	scanf("%d",&iValue);
	iRet = MultFact(iValue);
	printf("%d",iRet);
	return 0;
}